Calculator/Calculator/Calculator.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Not a nasty comment but just so you know, this is creating a dotnet core application at verion 3.1. We are currently working in dotnet 6 preview 4, but understand this is the sample, and dotnet core 3.1 has LTS (Long Term Support)

Reply:
OK - that's what the tutorial told me to put it to.  But happy to do next tutorial set in 6 preview 4

Reply:
Nope, do them all in what it recommends, better not to get tripped up